8.cytopick_analysis/scripts/feature_median_image_find.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+# import pillow and open cv
+import PIL
+import seaborn as sns
+import tifffile as tf
+from cytocherrypick import cherrypick
+from PIL import Image, ImageEnhance
+
+# In[2]:
+
+
+# parameters
+cell_type = "SHSY5Y"
+feature = "Nuclei_Texture_SumVariance_CorrGasdermin_3_01_256"
+
+
+# In[3]:
+
+
+# define directories
+# where the images are
+image_dir_path = pathlib.Path(
+    "/media/lippincm/18T/interstellar_data/70117_20230210MM1_Gasdermin514_CP_BC430856__2023-03-22T15_42_38-Measurement1/2.IC/"
+)
+# if path does not exist, create it
+image_dir_path.mkdir(parents=True, exist_ok=True)
```

Comment:
I am a bit confused here, why would you make a path to images if the path to the images don't exist? Wouldn't this just create an empty folder.

I would assume that you would be using `.resolve(strict=True)` here on the images_dir_path since you need to images to cytopick from.